src/sentry/models/avatars/__init__.py

```diff
@@ -1,9 +1,9 @@
+from ...users.models.users.user_avatar import UserAvatar
```

Comment:
Could we use absolute imports when we're crossing modules that are children of `sentry`?